### DISCUSSION TITLE
using a URL for a video

### DISCUSSION THREAD

**Discussion by ko56** [Goldenfreddy0703/Otaku#355] [Q&A] [answered]
Thanks for making Otaku available.  
Is there a way to watch a particular video, specified by a URL? For example,
[https://anime4i.com/watch-the-westward-season-5/ep-30-sv1.html](url)?
If not, is it possible to add `anime4i.com` as a provider, or something like that?
Thanks.

**Comment by Goldenfreddy0703** [comment]
Not really cause when it comes to embeds, we would have to add the url manually sense its takes encoding and decoding. Put that in our Otaku Suggestions and we can probably add anime4i in the future in our provider list

**Reply by ko56** [reply, marked as answer]
Thanks, I just did.
